ci/testnet-deploy.sh

```diff
@@ -148,21 +156,26 @@ trap shutdown EXIT INT
 
 set -x
 
+# Build a string to pass zone opts to $cloudProvider.sh: "-z zone1 -z zone2 ..."
+for val in "${zone[@]}"; do
+  zone_args="-z $zone_args $val"
```

Comment:
I think, this should be 
```
zone_args="$zone_args -z $val"
```
Similar change would be needed in testnet-manager.sh script.

Reply:
Done